src/node_file.cc

```diff
@@ -418,7 +418,10 @@ void FSReqWrap::Resolve(Local<Value> value) {
     Null(env()->isolate()),
     value
   };
-  MakeCallback(env()->oncomplete_string(), arraysize(argv), argv);
+  int len = arraysize(argv);
+  MakeCallback(env()->oncomplete_string(),
+               len == 2 && value->IsUndefined() ? 1 : len,
```

Comment:
Isn't the length check here redundant as it will always be 2? Why not just check the `undefined`-ness?